Cargo.toml

```diff
@@ -2,6 +2,43 @@
 resolver = "2"
 members = ["crates/*"]
 
-[patch.crates-io]
-# Warp fork with an extra commit to allow turning rejections into responses.
-warp = { git = 'https://github.com/vkgnosis/warp.git', rev = "87a91e24311b0ca6ed67408d2d302c569331dfec" }
+[workspace.dependencies]
+anyhow = "1"
+async-trait = "0.1"
+bigdecimal = "0.3"
+cached = { version = "0.34", default-features = false }
+chrono = { version = "0.4", default-features = false }
+clap = { version = "3", features = ["derive", "env"] }
+derivative = "2"
+ethcontract = { version = "0.21", default-features = false }
+ethcontract-generate = { version = "0.21", default-features = false }
+ethcontract-mock = { version = "0.21", default-features = false }
+futures = "0.3"
+gas-estimation = { git = "https://github.com/cowprotocol/gas-estimation", tag = "v0.7.1", features = ["web3_", "tokio_"] }
+hex = { version = "0.4", default-features = false }
+hex-literal = "0.3"
+itertools = "0.10"
+lazy_static = "1"
+maplit = "1"
+mockall = "0.11"
+num = "0.4"
+once_cell = "1"
+primitive-types = "0.10"
+prometheus = "0.13"
+prometheus-metric-storage = { git = "https://github.com/cowprotocol/prometheus-metric-storage", tag = "v0.4.0" }
+rand = "0.8"
+regex = "1"
+reqwest = "0.11"
+secp256k1 = "0.21"
+serde = { version = "1", features = ["derive"] }
+serde_json = "1"
+serde_with = { version = "1", default-features = false, features = ["macros"] }
+sqlx = { version = "0.6", default-features = false, features = ["runtime-tokio-native-tls"] }
```

Comment:
It turns out a `runtime-*` feature is needed to build `sqlx`. Without this, building individual crates with `cargo build -p $crate` fails.

For example, currently on main:
```
% cargo build -p database
   Compiling sqlx-rt v0.6.0
error: one of the features ['runtime-actix-native-tls', 'runtime-async-std-native-tls', 'runtime-tokio-native-tls', 'runtime-actix-rustls', 'runtime-async-std-rustls', 'runtime-tokio-rustls'] must be enabled
  --> /Users/nlordell/.cargo/registry/src/github.com-1ecc6299db9ec823/sqlx-rt-0.6.0/src/lib.rs:9:1
   |
9  | / compile_error!(
10 | |     "one of the features ['runtime-actix-native-tls', 'runtime-async-std-native-tls', \
11 | |      'runtime-tokio-native-tls', 'runtime-actix-rustls', 'runtime-async-std-rustls', \
12 | |      'runtime-tokio-rustls'] must be enabled"
13 | | );
   | |_^

error: could not compile `sqlx-rt` due to previous error
```